src/main.js

```diff
@@ -223,6 +224,18 @@ const CodingManagerApplication = new Lang.Class({
         });
 
         this._window.connect('notify::visible', Lang.bind(this, this._on_visibility_changed));
+        this._window.connect('map-event', Lang.bind(this, function() {
```

Comment:
The social bar has a similar workaround: https://github.com/endlessm/eos-social/blob/master/EosSocial/socialBarView.js#L494

Could you also use the same logic here? I'd like to avoid doing the same thing in two different ways.